CONTRIBUTING.md

```diff
@@ -122,8 +126,8 @@ visibility on the problem and work toward resolution.
 
 ## Closing issues
 
-You can either close issues manually by adding the fixing commit SHA1 to the issue
-comments or by adding the `Fixes` keyword to your commit message:
+The preferred way to close issues is by adding the `Fixes` keyword to your commit message.
+If your PR contains multiple commits then only one `Fixes` is required to be present the series, but multiple may be present if necessary.
```

Comment:
s/present the/present in the/
I'll fix up and re-push once the CIs have completed in order not to confuse them with the 'quick repush' issue.